core/src/main/java/org/ehcache/core/statistics/ValueStatistic.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright Terracotta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.ehcache.core.statistics;
+
+import org.terracotta.management.model.stats.StatisticType;
+
+import java.io.Serializable;
+
+public interface ValueStatistic<T extends Serializable> {
```

Comment:
if all the stat interfaces are moved to Ehcache, what is the plan for other projects that are also using them ?